release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2773,6 +2773,36 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.15.38
+[id="ocp-4-15-38_{context}"]
+=== RHSA-2024:8991 - {product-title} 4.15.38 bug fix and security update
+
+Issued: 13 November 2024
+
+{product-title} release 4.15.38, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:8991[RHSA-2024:8991] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2024:8994[RHSA-2024:8994] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.38 --pullspecs
+----
+
+[id="ocp-4-15-38-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, an invalid or unreachable identity provider (IDP) blocked updates to {hcp}. With this release, the `ValidIDPConfiguration` condition in the `HostedCluster` object now reports any IDP errors and these errors do not block updates of {hcp}. (link:https://issues.redhat.com/browse/OCPBUGS-44201[*OCPBUGS-44201*])
+
+* Previously, the Machine Config Operator (MCO) `vSphere `resolv-prepender` script used `systemd` directives that were not compatible with old boot image versions {product-title} 4. With this release, {product-title} nodes can scale with a boot image 4.13 or later, manual intervention, or an upgrade to a release with this fix. (link:https://issues.redhat.com/browse/OCPBUGS-42110[*OCPBUGS-42110*])
```

Comment:
I _think_ this should be:

```suggestion
* Previously, the Machine Config Operator (MCO) vSphere `resolv-prepender` script used systemd directives that were not compatible with old boot image versions of {product-title} 4. With this release, {product-title} nodes can scale with a boot image 4.13 or later, manual intervention, or an upgrade to a release with this fix. (link:https://issues.redhat.com/browse/OCPBUGS-42110[*OCPBUGS-42110*])
```
The final sentence is a little hard to parse to me. Might be that I'm reviewing too early in my day, but double check. :)